lib/util.js

```diff
@@ -0,0 +1,23 @@
+function getParameterByName(name) {
```

Comment:
Yeah I didn't write that myself either, parseUri is probably a safer alternative.